discordia/client/API.lua

```diff
@@ -72,7 +72,12 @@ function API:request(method, route, endpoint, payload)
 	end
 
 	if method:find('P') then
-		payload = payload and encode(payload) or '{}'
+		if isMulti then
+			reqHeaders[3][2] = 'multipart/form-data; boundary=Discordia'
```

Comment:
This is not good. There is no guarantee that the third header is always `content-type`.